### ISSUE TITLE
Keyboard example

### ISSUE THREAD

**Issue by chrisbuttery** [yoshuawuyts/choo#22]
Hey, nice work on `choo`!
I've just had a play with the [keyboard example](https://github.com/yoshuawuyts/choo#keyboard).

For my implementation I had to remove the ref to `keyboard` and rename 'app:print' to `print` in effects.
I am also accessing the payload value in the effect via `state`.

Am I `choo`ing wrong?

```js
const choo = require('choo')
const app = choo()

app.model({
  namespace: 'input',
  subscriptions: [
    function (send) {
      onkeypress = (e) => send('input:print', { payload: e.keyCode })
    }
  ],
  effects: {
    // state = type,payload
    // event = undefined
    print: (state, event) => console.log(`pressed key: ${state.payload}`)
  }
})

const mainView = (params, state, send) => {
  return choo.view`
    <main class="app"></main>
  `
}

app.router((route) => [
  route('/', mainView)
])

const tree = choo.start()
document.body.appendChild(tree)
```

**Comment by yoshuawuyts** [comment]
Oh yeah, I recently introduced namespaces which might mess with some of this. I should __really__ add unit tests soon haha. - Your code is looking fine; if it means the examples are broken: PRs definitely welcome for this! :sparkles:

**Comment by yoshuawuyts** [comment]
Closed by #28, thanks again! :D
